src/_common/Catalog/Catalog.cs

```diff
@@ -369,7 +369,7 @@ JsonValueKind.Number when element.TryGetDouble(out double doubleValue) => double
     /// </summary>
     private sealed class StyleJsonConverter : JsonConverter<Style>
     {
-        public override Style Read(ref Utf8JsonReader reader, Type _, JsonSerializerOptions options)
+        public override Style Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
```

Comment:
Add a file-specific suppression for the code analysis ID that Codacy is pointing out in 2 places for "Remove this unused method parameter 'options'." since these signature is inherited from a third-party package that we can't change.

Use the Codacy MCP server to verify that the suppression is working to silence it's issue reporting in Codacy.